services/core-web/common/utils/helpers.js

```diff
@@ -39,11 +39,17 @@ export const createItemMap = (array, idField) => {
 export const createItemIdsArray = (array, idField) => array.map((item) => item[idField]);
 
 export const createDropDownList = (array, labelField, valueField, isActiveField = false) => {
-  return array.map((item) => ({
+  const options = array.map((item) => ({
```

Comment:
Possible to use our helper for sorting a list of objects by a string attribute, `nullableStringSorter`?

Reply:
It appears like 'nullableStringSorter' is set up to accept different params than my intention here.

Reply:
I think `options = options.sort(nullableStringSorter("label"));` should do it, but np.